mungegithub/github/github.go

```diff
@@ -1150,3 +1150,45 @@ func (config *Config) ForEachIssueDo(fn MungeFunction) error {
 	}
 	return nil
 }
+
+// ListAllIssues grabs all issues matching the options, so you don't have to
+// worry about paging. Enforces some constraints, like min/max PR number and
+// having a valid user.
+func (config *Config) ListAllIssues(listOpts *github.IssueListByRepoOptions) ([]*github.Issue, error) {
```

Comment:
I didn't reuse because there's no way to select by label-- I wanted to let the github api do the work of deciding what matches the label.